utility/src/main/java/org/oppia/util/firebase/CrashLogger.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.util.firebase
+
+import java.lang.Exception
+
+/**
+ * Interface that enables exception logging.
+ */
+interface CrashLogger {
```

Comment:
Is this really a crash logger, or just an exception logger? A crash logger suggest it's specific to fatal crashes.